translations/da.po

```diff
@@ -51482,7 +51482,7 @@ msgid "\"The Additional Information service is used to retrieve cover images. \"
 msgstr ""
 
 #: /admin/config/ting/covers/addi?render=overlay
-msgid "\"URL to the Additional Information webservice, e.g. \"http://moreinfo.addi.dk/2.1/"
+msgid "\"URL to the Additional Information webservice, e.g. \"http://moreinfo.addi.dk/2.10"
```

Comment:
You updated the translation but you seem to have missed [the original text](https://github.com/ding2/ding2/blob/master/modules/ting_covers_addi/ting_covers_addi.admin.inc#L27) 😄 .